### ISSUE TITLE
Auto Save

### ISSUE THREAD

**Issue by momin-riyadh** [jsitor/jsitor#58]
This is an issue for **feature** request Auto save after every editing or every interval of time whenever changed detect snippets will be auto changed with auto-generated name.

**Comment by ashvin777** [comment]
Hi @momin-riyadh , Trying to understand the problem, 

I think what you are asking for is, ability to autosave the snippets whenever we do any changes in the code. 
I didn't understand fully "with auto-generated name.", can you please elaborate ?

Please check

**Comment by momin-riyadh** [comment]
Yes, simply auto save features and auto-generated new project name. Of course, the file or folder name will be editable.

**Comment by ashvin777** [comment]
What name will it be auto generated to ?

Can you share some example to help me understand this better ?

**Comment by momin-riyadh** [comment]
Yes, you can follow  https://codesandbox.io/
